db/migrate/20220705155001_create_users.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :name
       t.string :photo
       t.text :bio
-      t.integer :posts_counter, default: 0
+      t.integer :posts_counter
 
       t.timestamps
```

Comment:
- [ ] Please reference my earlier suggestion and add a default value of 0 to the `posts_counter`